dev-requirements.txt

```diff
@@ -3,6 +3,8 @@ nose-exclude==0.4.1
 mock==1.3.0
 coverage==3.7.1
 tox==2.1.1
+pathlib
+reFILE
```

Comment:
That could be done in an external script, but not inline in tox, and would not work on Windows.